packages/edit-navigation/src/components/menus-editor/index.js

```diff
@@ -15,14 +19,16 @@ export default function MenusEditor( { blockEditorSettings } ) {
 	const menus = useSelect( ( select ) => select( 'core' ).getMenus() );
 
 	const [ menuId, setMenuId ] = useState( 0 );
+	const [ stateMenus, setStateMenus ] = useState( false );
```

Comment:
Let's use `null` here instead of `false`. `null` is the more canonical way in JavaScript to represent unset or missing data and lets you use things like the `?.` and `??` operators.